lighthouse-core/test/results/sample_v2.json

```diff
@@ -3833,19 +3833,6 @@
             "target": "_blank",
             "rel": "nofollow",
             "outerHTML": "<a href=\"https://www.google.com/\" target=\"_blank\" rel=\"nofollow\">"
-          },
-          {
```

Comment:
> a "internal link is ok" link has been in our external-anchors audit results for a bit. but.. like the link says.. it's okay.

this is the only diff you were referring to with that quote, right?

Reply:
yup!